src/include/user/abi_dbg.h

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ *
+ * Author: Marcin Maka <marcin.maka@linux.intel.com>
+ */
+
+/**
+ * \file include/user/abi_dbg.h
+ * \brief ABI definitions for debug interfaces accessed by user space apps.
+ * \author Marcin Maka <marcin.maka@linux.intel.com>
+ *
+ * Follows the rules documented in include/user/abi.h.
+ */
+
+#include <kernel/abi.h>
+
+#ifndef __USER_ABI_DBG_H__
+#define __USER_ABI_DBG_H__
+
+#define SOF_ABI_DBG_MAJOR 3
+#define SOF_ABI_DBG_MINOR 14
+#define SOF_ABI_DBG_PATCH 0
```

Comment:
Is 'debug abi' the best name for this? We have folders that are named by-abi-user (kernelspace or userspace), but ABI version symbols are named by-feature (generic vs debug).

Reply:
Nvm, I see it's going to be more granular for user, versions are in `sof_ipc_user_abi_version `